CybeReason/malop/_meta/fields.yml

```diff
@@ -0,0 +1,64 @@
+cybereason.cef.version:
+  name: cybereason.cef.version
+  description: "The version of the CEF message"
+  type: text
```

Comment:
I suggest to use `keyword` when possible

ATM `ingest` does not propagate types to elastic but it could be different in the future
![Screenshot from 2022-05-06 11-25-20](https://user-images.githubusercontent.com/8960084/167105423-f1439716-814a-4f41-a450-97a1efd9575c.png)

Reply:
applied